gazebo_ros/scripts/spawn_model

```diff
@@ -288,6 +288,7 @@ if __name__ == "__main__":
         print usage()
     else:
         print("spawn_model script started") # make this a print incase roscore has not been started
+        rospy.init_node('spawn_model')
```

Comment:
got it. added issue #301. I hope one day to enable unit / regression tests for this stack.